packages/node-integration-tests/utils/index.ts

```diff
@@ -68,48 +75,69 @@ export const parseEnvelope = (body: string): Array<Record<string, unknown>> => {
 /**
  * Intercepts and extracts up to a number of requests containing Sentry envelopes.
  *
- * @param url The url the intercepted requests will be directed to.
+ * @param config The url, server instance and the nock scope.
  * @param count The expected amount of requests to the envelope endpoint. If
  * the amount of sentrequests is lower than`count`, this function will not resolve.
  * @param method The method of the request. Defaults to `GET`.
+ * @param endServer: Whether to stop the server after the requests have been intercepted.
  * @returns The intercepted envelopes.
  */
 export const getMultipleEnvelopeRequest = async (
-  url: string,
+  config: TestServerConfig,
   count: number,
   method: 'get' | 'post' = 'get',
+  endServer: boolean = true,
```

Comment:
Now that there are three options, can we pass this in as an options obj (2nd arg)?